syntax = "proto3";
option go_package = ".";

package translate;

service Translator {
		rpc Translate(TranslationRequest) returns (TranslationResponse) {}
	}

message TranslationRequest {
		string text = 1;
		string fromLang = 2;
		string toLang = 3;
	}

message TranslationResponse {
		int32 status = 1;
		string text = 2; 
	}
